docs/en/compute/AWS-EMR.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: global
+title: Running Alluxio on EMR
```

Comment:
Running EMR with Alluxio?

Reply:
Technically we're running Alluxio on an EMR cluster. Running 'EMR with Alluxio' could mean that the Alluxio cluster is remote to EMR not actually alongside EMR. Also doesn't convey that we work with some of the native EMR features like Glue MetaStore and Auto-Scaling